### PR TITLE
Change location for loading thebe and make it always load the latest version.

### PR DESCRIPTION
**Pull request by moorepants**

### CODE REVIEW

**Comment by moorepants** [comment]
@narrator0 can you review? If you don't think using latest is smart we can manually increment the version number.

**Comment by narrator0** [comment]
I would go for `0.6.0` since I don't think it is stable enough. Don't want libretexts pages to fail because of this.

**Comment by celine168** [comment]
I agree with 0.6.0.

File: src/scripts/activateThebelab.js [hunk 1]
```diff
@@ -67,7 +67,7 @@ const activateThebelab = (config, detectLanguage = true) => {
       mergeConfig = Object.assign(mergeConfig, getConfig(language));
     }
 
-    loadScript('https://unpkg.com/thebelab@0.5.1/lib/index.js')
+    loadScript('https://unpkg.com/thebe@latest/lib/index.js')
```
Comment (celine168) [inline src/scripts/activateThebelab.js]:
```suggestion
    loadScript('https://unpkg.com/thebe@0.6.0/lib/index.js')
```

**Review by celine168 — CHANGES_REQUESTED** [review]

**Comment by moorepants** [comment]
I committed the suggestion.

**Review by celine168 — APPROVED** [review]

**Comment by moorepants** [comment]
This is good to go in @LibreTexts/jupyter-team .

**Pull request merged**